src/combobox/ComboBox.ts

```diff
@@ -89,19 +83,18 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 		this.invalidate();
 	}
 
+	private _getMenuId() {
+		return `${this._idBase}-menu`;
+	}
+
 	private _getResultLabel(result: any) {
 		const { getResultLabel } = this.properties;
 
-		return getResultLabel ? getResultLabel(result) : result;
+		return getResultLabel ? getResultLabel(result) : `${result}`;
```

Comment:
Could this just be `String(result)`?

Reply:
Template strings are the most optimal way to do string conversion now.